From 46ace441aaa6b9c6ba3060fcf4fe630a2caa0b11 Mon Sep 17 00:00:00 2001
From: Sebastian Doell <sdoell@hilscher.com>
Date: Thu, 16 Sep 2021 12:28:11 +0000
Subject: [PATCH] set_boot_image_size_to_64MB

---
 common/bootm.c | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/boot/bootm.c b/boot/bootm.c
index db4362a643..417800060c 100644
--- a/boot/bootm.c
+++ b/boot/bootm.c
@@ -33,6 +33,12 @@
 #include <bootm.h>
 #include <image.h>
 
+#ifdef CONFIG_SYS_BOOTM_LEN
+#undef CONFIG_SYS_BOOTM_LEN
+#endif
+/* default to 64MB, otherwise fit_check_sign will fail with "image to large" */
+#define CONFIG_SYS_BOOTM_LEN 0x4000000
+
 #ifndef CONFIG_SYS_BOOTM_LEN
 /* use 8MByte as default max gunzip size */
 #define CONFIG_SYS_BOOTM_LEN	0x800000
-- 
2.17.1

